dask_jobqueue/core.py

```diff
@@ -607,6 +625,46 @@ def _new_worker_name(self, worker_number):
             cluster_name=self._name, worker_number=worker_number
         )
 
+    def _get_worker_security(self, security):
```

Comment:
This function could be extended to also handle cases where file certificates are outside the shared directory path. For now it only handles creating temporary files. Unfortunately the files will survive until garbage collected (which is probably longer than needed). Security is passed as an argument because the initialization of self done is too late by the super class.